src/Compilers/CSharp/Portable/Syntax/NullableDirectiveMap.cs

```diff
@@ -40,14 +57,11 @@ private NullableDirectiveMap(ImmutableArray<(int Position, bool? State)> directi
             _isGeneratedCode = isGeneratedCode;
         }
 
-        /// <summary>
-        /// Returns true if the `#nullable` directive preceding the position is
-        /// `enable` or `safeonly`, false if `disable`, and null if no preceding directive,
-        /// or directive preceding the position is `restore`.
-        /// </summary>
-        internal bool? GetDirectiveState(int position)
+        internal NullableDirective GetDirectiveState(int position)
```

Comment:
That doc comment looked useful. Why delete it?

Reply:
Previously we used a tuple, so documentation for the return type was on this method (and another similar one). Now we return a dedicated type and the documentation was moved to that type.

---
In reply to: [295079655](https://github.com/dotnet/roslyn/pull/36464#discussion_r295079655) [](ancestors = 295079655)

Reply:
The information is still there, just moved to the declaration of `NullableDirective`.